code/chart.js

```diff
@@ -1,4 +1,21 @@
-//DOM-selector for the canvas 👇
-const ctx = document.getElementById('chart').getContext('2d')
+const ctx = document.getElementById("chart").getContext("2d");
+const allProjects = 19;
 
-//"Draw" the chart here 👇
+const drawChart = (finishedProjects) => {
+  const config = {
+    type: "doughnut",
+    data: {
+      labels: ["Finished projects", "Projects left"],
+      datasets: [
+        {
+          label: "Technigo projects",
+          data: [finishedProjects, allProjects - finishedProjects],
+          backgroundColor: ["rgb(204, 194, 193)", "rgb(140, 100, 114)"],
```

Comment:
Nice that you instead are naming it allProjects - finishedProjects instead of amount. Makes more sense!